best-practices/tagging/index.html

```diff
@@ -342,6 +354,12 @@ <h4>Line numbers</h4>
 					<dd>
 						<ul class="nomark">
 							<li><code>id</code>, <code>linenum</code>, <code>prose</code>, <code>poetry</code></li>
+								<ul class = "nomark">
+									<li><code>id</code>- used to denote the actual line number in print to assist software with utilizing the information without the need for backtranslation.</li>
```

Comment:
Just a general note that em dashes are usually best to offset descriptions. You can add these quickly with the entity &amp;#8212;

Reply:
Changed all hyphens being used to offset descriptions to &#8212;.